homeassistant/components/webostv/config_flow.py

```diff
@@ -129,11 +129,37 @@ async def async_step_ssdp(self, discovery_info: ssdp.SsdpServiceInfo) -> FlowRes
         self._uuid = uuid
         return await self.async_step_pairing()
 
+    async def async_step_reauth(self, entry_data: Mapping[str, Any]) -> FlowResult:
+        """Perform reauth upon an WebOsTvPairError."""
+        self._host = entry_data[CONF_HOST]
+        self._entry = self.hass.config_entries.async_get_entry(self.context["entry_id"])
+        return await self.async_step_reauth_confirm()
 
-class OptionsFlowHandler(config_entries.OptionsFlow):
+    async def async_step_reauth_confirm(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Dialog that informs the user that reauth is required."""
+        assert self._entry is not None
+
+        if user_input is not None:
+            try:
+                client = await async_control_connect(self._host, None)
+            except WebOsTvPairError:
+                return self.async_abort(reason="error_pairing")
+            except WEBOSTV_EXCEPTIONS:
+                return self.async_abort(reason="reauth_unsuccessful")
+            else:
```

Comment:
We can remove this `else:` and outdent below.

Reply:
Thanks and to think about that just before submitting this PR I saw the one about removing `else` from integrations, but still didn't notice this occurrence 😄